sdk/template/template/karma.conf.js

```diff
@@ -60,7 +61,10 @@ module.exports = function(config) {
       "APPCONFIG_TEST_SETTING_EXPECTED_VALUE",
       "AZURE_CLIENT_ID",
       "AZURE_CLIENT_SECRET",
-      "AZURE_TENANT_ID"
+      "AZURE_TENANT_ID",
+      "TABLES_SAS_CONNECTION_STRING",
+      "RECORDINGS_RELATIVE_PATH",
```

Comment:
Will all packages need to add this env var configuration? What is it used for? Seems a little weird to load a value from the recorder, stuff it into env, and then presumably read it again in the browser?

Reply:
`RECORDINGS_RELATIVE_PATH` is required for the recorder to know where to dump the files, the others will drop away when I remove the extra tests.

Reply:
We need to let the proxy tool know where to save the recordings, the only way is to let it be an env variable and load in the browser.
I was hoping, this wouldn't be a burden once we migrate to the unduplicated karma conf from dev-tool.

Do you have any suggestions?

Reply:
That function calls `process.cwd()` to get the project path and that'd only be available in node.
We've to pass it on to the browser environment as well, which can be possible only through karma.

Reply:
Try to move to the model where the path is obtained at the dev-tool itself.
To avoid this excess code in karma.conf.

Reply:
We're keeping those changes out-of-scope for this PR though, right?

Reply:
Yes, can you log an issue with the details and put it in the epic?

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/19386.

Reply:
Considering to move the call at least to the dev-tool test:browser command.
I'm afraid of the experience with debugging using dotnet tool or docker independently if we move both the pieces.